plugins/exporter.koplugin/formats/json.lua

```diff
@@ -0,0 +1,32 @@
+local json = require("json")
+
+local JsonExporter = require("formats/base"):new {
+    name = "json",
+}
+
+function JsonExporter:exportOne(t)
```

Comment:
fix the function, it should be `export` instead of `exportOne`

Reply:
Done.